internal/cli/flags.go

```diff
@@ -0,0 +1,91 @@
+package cli
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/CDCgov/data-exchange-csv/cmd/internal/constants"
+	"github.com/CDCgov/data-exchange-csv/cmd/internal/models"
+)
+
+func ParseFlags() models.FileValidateInputParams {
+	var receivedFile string
+	var destination string
+	var debug bool
+	var logToFile bool
+	var transform bool
+	var configFile string
+
+	flag.StringVar(&receivedFile, "fileURL", "", "[Required] The path to the file that will be validated")
+	flag.BoolVar(&debug, "debug", false, "[Optional] enable debug mode to generate debug-level logs")
+	flag.BoolVar(&logToFile, "log-file", false, "[Optional] If true, logs will be written to logs/validation.json, default is stdout")
+	flag.BoolVar(&transform, "transform", false, "[Optional] If true, the validated rows will be transformed to JSON")
+	flag.StringVar(&destination, "destination", "", "[Required] The URL to the folder where validation/transformation results will be stored")
+	flag.StringVar(&configFile, "config", "", "[Optional] The URL to the config.json. If provided overrides auto-detection of encoding, and separator")
+
+	flag.Parse()
+
+	if receivedFile == "" || destination == "" {
```

Comment:
Cosmin gave a suggestion to trim arguments before testing equality on empty string.

Reply:
Resolved in https://github.com/CDCgov/data-exchange-csv/pull/12/commits/9971ebe6112dadf3e70d1b269c89a1ecb32efa25